### ISSUE TITLE
Copy page not work

### ISSUE THREAD

**Issue by suxilog** [infoportugal/wagtail-modeltranslation#283]
**Issue Summary**
Set WAGTAILMODELTRANSLATION_TRANSLATE_SLUGS to False, copy any page, it shows
```
ValidationError at /admin/pages/10/copy/
{'slug': ['This slug is already in use']}
```

**Steps to Reproduce**
1. Set `WAGTAILMODELTRANSLATION_TRANSLATE_SLUGS = False`
2. Add a new homepage, copy it and change the slug, then click copy this, it will show ValidationError.

When set WAGTAILMODELTRANSLATION_TRANSLATE_SLUGS to True,  the copy page worked successful.I think it should display only one slug field in copy page when translate slug set to False.

wagtail==2.9 and 2.8
Django==2.2.12
wagtail-modeltranslation==0.10.13

**Comment by jwsmithers** [comment]
While this wasn't my issue, it did _finally_ fix my error by providing the  option `WAGTAILMODELTRANSLATION_TRANSLATE_SLUGS` which did finally fix my error. I haven't seen this in the documentation anywhere.
 So thanks for that 🙂

**Comment by Ramblurr** [comment]
This is affecting me too. 

### Steps to Reproduce

* In an existing site, navigate to the pages list
* Click More > Copy
* Attempt to choose another page by clicking "Choose another page" 

Result: nothing happens

Check the browser's console: and see the error `ReferenceError: createPageChooser is not defined`

I confirmed this issue using a fresh wagtail project and adding wagtail-modeltranslation.

* Django 3.0.6
* Wagtail 2.9
* Python 3.7.6

**Comment by juan0tron** [comment]
Having the same issue here. @Ramblurr - if you're still blocked by the JS Page Chooser issue, you can override the template `templates/modeltranslation_copy.html` in your project to temporarily fix it until your update is available on pypi.

**Comment by juan0tron** [comment]
Just submitted PR #284 - that will fix this issue for anyone who specifically needs `WAGTAILMODELTRANSLATION_TRANSLATE_SLUGS` to be `False` in their project.

Currently using:

- Django 2.2.6
- Wagtail 2.9
- Python 3.6.8

**Comment by DiogoMarques29** [comment]
Fixed in #284
